packages/registrar_client/tests/integration_tests/farm.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, test, expect } from "@jest/globals";
+import tweetnacl from "tweetnacl";
+import base64 from "base64-js";
+import { RegistrarClient } from "../../src/client/client";
+
+describe("test farm module", () => {
+  const keyPair = tweetnacl.sign.keyPair();
+  const privateKey = base64.fromByteArray(keyPair.secretKey);
+
+  const client = new RegistrarClient({ baseURL: "http://registrar:8080/v1", privateKey: privateKey });
+
+  let twinID = 1;
+  let farmID = 1;
+
+  test("create farm", async () => {
```

Comment:
I think we can add some more test cases: 
- create a farm with no existing twin ID
- create a farm with an already existing farm name/id
- create a farm with an invalid name